python_modules/libraries/dagster-duckdb-pyspark/dagster_duckdb_pyspark/duckdb_pyspark_type_handler.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+
+import duckdb
+import pyspark
+from dagster_duckdb import DbTypeHandler
+
+from dagster import InputContext, OutputContext
+from dagster import _check as check
+
+
+class DuckDBPySparkTypeHandler(DbTypeHandler[pyspark.sql.DataFrame]):
+    """Stores PySpark DataFrames in DuckDB.
+
+    **Note:** This type handler can only store outputs. It cannot currently load inputs.
```

Comment:
Why is this the case? I think the pandas use case is more common, so this isn't the end of the world or anything, but ideally all of our type handlers would handle both input and output.

I think this should be achievable with just a `spark.createDataFrame(df)`, where `df` is a pandas DataFrame, but there might be a more direct way of doing this